talk/objectorientation/constructors.tex

```diff
@@ -241,7 +243,7 @@
   \begin{exampleblock}{Practically}
     \begin{cppcode}
       struct BaseClass {
-        int a{5};
+        int a{5}; // also possible: int a = 5; 
```

Comment:
We could generally consider having a slide on the different ways to initialize variables.

Reply:
We could. Do you want to create on issue?